packages/wpk/test/deb_upgrade/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM ubuntu:22.04
+RUN apt update && apt install lsb-release procps curl pip -y
```

[Based on best practice](https://docs.docker.com/build/building/best-practices/#sort-multi-line-arguments)
```suggestion
RUN apt update && apt install -y \
    lsb-release \
    procps \
    curl \
    pip \
    && rm -rf /var/lib/apt/lists/*
```

---

Addressed with https://github.com/wazuh/wazuh/pull/31575/commits/a2037a5cfd888dedda12c0f21485235950a8f169